src/components/SubCategoryDetails/SubCategoryDetails.scss

```diff
@@ -0,0 +1,45 @@
+@import '~style/variables';
+
+.container {
+  display: flex;
+  flex-direction: column;
```

Comment:
You don't need flex here, the children are all just block level elements going down the page.
Setting these three have no effect, so can be removed:
```css
display: flex;
flex-direction: column;
width: 100%;
```